marklogic-data-hub/src/main/java/com/marklogic/hub/hubcentral/migration/FlowMigrator.java

```diff
@@ -149,6 +149,11 @@ public void migrateFlows() {
             ObjectNode newSteps = nodeFactory.objectNode();
             for (Map.Entry<String, Step> entry : steps.entrySet()) {
                 Step step = entry.getValue();
+                JsonNode stepJson = step.getStepJson();
+                if (stepJson.has("stepId") && stepJson.size() == 1) {
```

Comment:
Argh, sorry, I didn't realize the ObjectNode wasn't available. I think it's fine then to add stepId to the Step class. FlowMigrator can just say - if stepId is populated, then I don't need to migrate it. Then you don't need the stepJson JsonNode in Step.

Reply:
ok, I modified it to test for stepId not null.